changes/ana_add-no-mainnet-for-this-address-found-to-sign-in

```diff
@@ -0,0 +1 @@
+[Changed] [#3553](https://github.com/cosmos/lunie/pull/3553) Adds check in SessionSignIn to verify the signin address belongs to an existing mainnet. Otherwise displays an error message @Bitcoinera
```

Comment:
how can we tell if it belongs to a mainnet?

Reply:
I mean, maybe I should rephrase this. 

Right now we don't have in place a way to tell whether the account was created in mainnet or in testnet (which I don't think is needed), so there is no way to tell this.

What I wanted to mean here is, it checks if there is an existing mainnet for the network family the address belongs to. If there isn't any (like nowadays in Regen ,for instance), it displays the error message. You need to check the testnet checkbox.

Reply:
> You need to check the testnet checkbox.

could we "check it" automatically?

Reply:
Actually this is how it should be and it is something I have noticed in our flow which should be fixed

Reply:
Oh, yes. But @faboweb said it was better to let users clearly know when they are entering a testnet. Maybe we could just drop a brief warning instead throw a validation error?

Reply:
i'm talking about the situation where a user selects a network (gaia testnet, for eg.), then clicks sign in, then enters a cosmos address and ends up on mainnet. if there is a testnet selected, the checkbox should be selected too.